src/one_dragon_qt/view/installer_interface.py

```
@@ -1,919 +1,213 @@
+        from pathlib import Path
 
-        # 如果是高级模式，检查所有步骤的状态
-        if self.is_advanced_mode:
-            for step in self.install_steps:
-                step.check_status()
+        from PySide6.QtWidgets import QApplication
+
+        launcher_path = Path(os_utils.get_work_dir()) / 'OneDragon-Launcher.exe'
+        if not launcher_path.exists():
+            return
+        # 用 os.startfile 而非 shell 拼接，避免安装路径含 & 等字符时被 cmd 解释
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)